examples/kyb-app/src/components/atoms/Stepper/components/atoms/Breadcrumb/hooks/useBreadcrumbElement.ts

```diff
@@ -0,0 +1,13 @@
+import { useBreadcrumb } from '@app/components/atoms/Stepper/components/atoms/Breadcrumb/hooks/useBreadcrumb';
+import { BreadcrumbElements } from '@app/components/atoms/Stepper/components/atoms/Breadcrumb/types';
+
+export const useBreadcrumbElement = <TProps>(
```

Comment:
If this hook is supposed to encapsulate business logic let's starting applying the new naming convention.
```suggestion
export const useBreadcrumbLogic = <TProps>(
```